rtest.xml

```diff
@@ -2,10 +2,14 @@
*0.TestTupleTie*:*0.TestUniqueByKey*:*1.TestUniqueByKey*:*2.TestUniqueByKey*:*3.TestUniqueByKey*:*4.TestUniqueByKey*:*5.TestUniqueByKey*:*UniqueByKeyTests.TestUniqueByKeyDispatchExplicit*:*1.TestUniqueCopy*:*2.TestUniqueCopy*:*4.TestUniqueCopy*:*5.TestUniqueCopy*:*0.TestUniqueCopyByKey*:*1.TestUniqueCopyByKey*:*2.TestUniqueCopyByKey*:*3.TestUniqueCopyByKey*:*4.TestUniqueCopyByKey*:*5.TestUniqueCopyByKey*:*0.TestUniqueCopyByKeyToDiscardIterator*:*1.TestUniqueCopyByKeyToDiscardIterator*:*2.TestUniqueCopyByKeyToDiscardIterator*:*3.TestUniqueCopyByKeyToDiscardIterator*:*4.TestUniqueCopyByKeyToDiscardIterator*:*5.TestUniqueCopyByKeyToDiscardIterator*:*1.TestUniqueCopyToDiscardIterator*:*2.TestUniqueCopyToDiscardIterator*:*4.TestUniqueCopyToDiscardIterator*:*5.TestUniqueCopyToDiscardIterator*:*0.TestUniqueCount*:*1.TestUniqueCount*:*2.TestUniqueCount*:*3.TestUniqueCount*:*4.TestUniqueCount*:*5.TestUniqueCount*:*UniqueTests.TestUniqueDevice/UniqueTests.TestUniqueDevice*:*UniqueIntegralTests.TestUniqueDevice/UniqueIntegralTests.TestUniqueDevice*:*UniqueTests.TestUniqueDispatchExplicit*:*0.TestUniversalAllocateUnique*:*0.TestVectorAssignFromDeviceVector*:*0.TestVectorBinarySearchDescending*:*1.TestVectorBinarySearchDescending*:*2.TestVectorBinarySearchDescending*:*0.TestVectorBinarySearchDiscardIterator*:*1.TestVectorBinarySearchDiscardIterator*:*2.TestVectorBinarySearchDiscardIterator*:*0.TestVectorLowerBoundDescending*:*1.TestVectorLowerBoundDescending*:*2.TestVectorLowerBoundDescending*:*8.TestVectorLowerBoundDescendingSimple*:*0.TestVectorLowerBoundDiscardIterator*:*1.TestVectorLowerBoundDiscardIterator*:*2.TestVectorLowerBoundDiscardIterator*:*8.TestVectorRangeInsertSimple*:*0.TestVectorUpperBoundDescending*:*1.TestVectorUpperBoundDescending*:*2.TestVectorUpperBoundDescending*:*0.TestVectorUpperBoundDiscardIterator*:*1.TestVectorUpperBoundDiscardIterator*:*2.TestVectorUpperBoundDiscardIterator*:*ZipIterator32BitTests.TestZipIteratorCopyAoSToSoA*:*0.TestZipIteratorReduceByKey*:*1.TestZipIteratorReduceByKey*:*2.TestZipIteratorReduceByKey*:*3.TestZipIteratorReduceByKey*:*0.TestZipIteratorScan*:*1.TestZipIteratorScan*:*2.TestZipIteratorScan*:*3.TestZipIteratorScan*:*4.TestZipIteratorScan*:*5.TestZipIteratorScan*:*6.TestZipIteratorScan*:*7.TestZipIteratorScan*:*ZipIteratorStableSortTests/0.TestZipIteratorStableSort*:*ZipIteratorStableSortTests/1.TestZipIteratorStableSort*:*ZipIteratorStableSortTests/2.TestZipIteratorStableSort*:*ZipIteratorStableSortTests/3.TestZipIteratorStableSort*:*ZipIteratorStableSortByKeyTests/0.TestZipIteratorStableSort*:*ZipIteratorStableSortByKeyTests/1.TestZipIteratorStableSort*:*ZipIteratorStableSortByKeyTests/2.TestZipIteratorStableSort*:*1.TransformIteratorReduce*:*2.TransformIteratorReduce*:*5.TransformIteratorReduce*:*6.TransformIteratorReduce*:*7.TransformIteratorReduce*:*0.TransformScan*:*1.TransformScan*:*2.TransformScan*:*3.TransformScan*:*4.TransformScan*:*5.TransformScan*:' ctest --output-on-failure -I 178,294"></var>
```

Comment:
I just have two quick questions:
1. I noticed there is a dash '-' character in the middle of this string, it's right after "*RandomTests.erfcinvFunction*:", so if you search for that you should find it. Just wanted to make sure it was intentionally placed.
2. I'm wondering if there's any way we could put one family of tests per line here (eg. put all the "TransformScan" regexes on one line) - it would make the file a little more readable in case we have to modify it in the future. Do you think it would it be possible to do that and then just eliminate all the newline characters in the Python script at the point where you retrieve the data from the XML attribute?

Reply:
For point 2.) I see what you mean. And certainly that is a possible way of doing it, although it would require some modification of rtest.py

Reply:
Ok, thanks, it was just an idea. I think we can leave this as is for now, since it probably needs to get in soon. I think it looks ready to go.